client/data/marketplace/use-wpcom-plugins-query.ts

```diff
@@ -47,16 +60,12 @@ export const useWPCOMPlugins = (
 	tag?: string,
 	{ enabled = true, staleTime = BASE_STALE_TIME, refetchOnMount = true }: UseQueryOptions = {}
 ): UseQueryResult => {
-	return useQuery(
-		getCacheKey( type + searchTerm + tag + `${ enabled ? 'enabled' : 'disabled' }` ),
-		() => fetchWPCOMPlugins( type, searchTerm, tag ),
-		{
-			select: ( data ) => normalizePluginsList( data.results ),
-			enabled: enabled,
-			staleTime: staleTime,
-			refetchOnMount: refetchOnMount,
-		}
-	);
+	return useQuery( ...getFetchWPCOMPlugins( enabled, type, searchTerm, tag ), {
```

Comment:
Needed to extract the cacheKey and fetch functions from the hook so that they can be used inside the node middleware to prefetch the data.